apps/marketing/src/components/(marketing)/widget.tsx

```diff
@@ -346,7 +346,7 @@ export const Widget = ({ className, children, ...props }: WidgetProps) => {
                   {signatureText && (
                     <p
                       className={cn(
-                        'text-foreground text-4xl font-semibold [font-family:var(--font-caveat)]',
+                        'text-foreground truncate text-4xl font-semibold [font-family:var(--font-caveat)]',
```

Comment:
This issue fixes the text issue on https://documenso.com

**Before:**

<img width="1024" alt="Screenshot 2024-03-12 at 13 21 59" src="https://github.com/documenso/documenso/assets/23498248/c4793d99-0340-459c-91ff-672f2977f3b5">

**After**

<img width="1030" alt="Screenshot 2024-03-12 at 13 22 17" src="https://github.com/documenso/documenso/assets/23498248/adb83888-4346-4055-a540-112ccd08d3ec">

Reply:
Nice one!

Should we truncate and add a right margin to the text near the "Sign" button? What do you think?

See a before and after in the pics below.

![CleanShot 2024-04-03 at 09 20 56](https://github.com/documenso/documenso/assets/25515812/ee989559-f0e8-4215-9009-d2b6d5eb5a49)
![CleanShot 2024-04-03 at 09 22 00](https://github.com/documenso/documenso/assets/25515812/d7b5faa7-2ec4-4da8-9a17-2f64a1dc082c)

Reply:
I second with you on this and Fixed it in the commit below 🚀 

<img width="568" alt="Screenshot 2024-04-04 at 00 15 47" src="https://github.com/documenso/documenso/assets/23498248/6a073339-efd4-4658-acdd-4a898ef12854">